site/docs/reference/Connectors/capture-connectors/hubspot-native.md

```diff
@@ -0,0 +1,116 @@
+* [Owners](https://developers.hubspot.com/docs/api/crm/owners)
+* [Properties](https://developers.hubspot.com/docs/api/crm/properties)
+
+The following data resources are supported for pro accounts:
+
+* [Feedback Submissions](https://developers.hubspot.com/docs/api/crm/feedback-submissions)
+* [Marketing Emails](https://developers.hubspot.com/docs/api/marketing/marketing-email)
+* [Workflows](https://legacydocs.hubspot.com/docs/methods/workflows/v3/get_workflows)
+
+## Prerequisites
+
+There are two ways to authenticate with Hubspot when capturing data: using OAuth2, and manually, with a private app access token.
+Their prerequisites differ.
+
+OAuth is recommended for simplicity in the Flow web app;
+the access token method is the only supported method using the command line.
+
+### Using OAuth2 to authenticate with Hubspot in the Flow web app
+
+* A Hubspot account
+
+### Configuring the connector specification manually
+
+* A Hubspot account
+
+* The access token for an appropriately configured [private app](https://developers.hubspot.com/docs/api/private-apps) on the Hubspot account.
+
+#### Setup
+
+To create a private app in Hubspot and generate its access token, do the following.
+
+1. Ensure that your Hubspot user account has [super admin](https://knowledge.hubspot.com/settings/hubspot-user-permissions-guide#super-admin) privileges.
+
+2. In Hubspot, create a [new private app](https://developers.hubspot.com/docs/api/private-apps#create-a-private-app).
+
+   1. Name the app "Estuary Flow," or choose another name that is memorable to you.
+
+   2. Grant the new app **Read** access for all available scopes.
+
+   3. Copy the access token for use in the connector configuration.
+
+## Configuration
+
+You configure connectors either in the Flow web app, or by directly editing the catalog specification file.
+See [connectors](../../../concepts/connectors.md#using-connectors) to learn more about using connectors. The values and specification sample below provide configuration details specific to the Hubspo-native connector.
+
+#### Endpoint
+
+The following properties reflect the access token authentication method.
+
+| Property | Title | Description | Type | Required/Default |
+|---|---|---|---|---|
+| **`/credentials`** | Private Application | Authenticate with a private app access token | object | Required |
+| **`/credentials/access_token`** | Access Token | HubSpot Access token. | string | Required |
+| **`/credentials/credentials_title`** | Credentials | Name of the credentials set | string | Required, `"Private App Credentials"` |
+| **`/start_date`** | Start Date | UTC date and time in the format 2017-01-25T00:00:00Z. Any data before this date will not be replicated. | string | Required |
+| `/subscription_type` | Your HubSpot account subscription type | Some streams are only available to certain subscription packages, we use this information to select which streams to pull data from. | string | `"starter"` |
```

Comment:
Sounds like this should be an enum not a free form string

Reply:
Fixed [here](951a289892c088058e124aeda6e39585332e4e3a)

Reply:
I meant that we should make this field in the connector a dropdown not a string if there is a set of discrete options. For instance if the options are ['starter', 'pro'] we shouldn't allow users to input whatever

Reply:
Now looking at the endpoint config for the new connector, this field doesn't exist anymore

Reply:
@jonwihl Oh yeah, this config was probably used before adding the "optional_scopes" param. This option allows hubspot to ignore non-approved scopes inside this category, so the API returns a empty result but allow validation, going to remove that on both of them